include/tracewrap.h

```diff
@@ -43,6 +43,9 @@ void qemu_trace_finish(uint32_t exit_code);
 OperandInfo * load_store_reg(target_ulong reg, target_ulong val, int ls);
 OperandInfo * load_store_mem(target_ulong addr, target_ulong val, int ls, int len);
 
+void *state(void);
```

Comment:
all qemu_trace functions should be prefixed with `qemu_trace_`

Note: the `load_store_reg` above, are exception to the rule, mostly because they shouldn't be here at all.